src/System.Net.Http/src/System/Net/Http/CurlHandler/CurlHandler.SslProvider.Linux.cs

```diff
@@ -21,6 +21,8 @@ internal partial class CurlHandler : HttpMessageHandler
     {
         private static class SslProvider
         {
+            private const string RequiredOpenSSLBackendName = "OpenSSL/1.0";
```

Comment:
Interop.Http.OpenSsl10Description is what is used in this class (line 55/57).  Consider setting this const equal to that const, or just using the existing one.

Reply:
Ah, I'll just use the existing ones.  Thanks.